RIOT-OS/RIOT
# gh_workflows: adapt to GNRC LoRaWAN Release Test env variables

### Contribution description
This PR fixes the invocation of `11-lorawan`. The keys were renamed in https://github.com/RIOT-OS/Release-Specs/pull/201, so the current keys are not applied. Therefore the test fails.

This probably needs backport.

### Testing procedure

Please check the variables match the ones exposed in https://github.com/RIOT-OS/Release-Specs/pull/201. We should also trigger this tests from GH Actions.



### Issues/PRs references
https://github.com/RIOT-OS/Release-Specs/pull/201
